plugins/modules/grafana_user.py

```diff
@@ -272,11 +272,12 @@ def main():
     # search user by login
     target_user = grafana_iface.get_user_from_login(login)
```

Comment:
`target_user` should be renamed to express that it is the user currently existing (or not) on grafana

Reply:
Maybe we could name it `stored_user` or `stored_grafana_user` ?

Reply:
`actual_grafana_user` ?

Reply:
Seems OK for me.